target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8068-cryptid-common.dtsi

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: GPL-2.0 OR MIT
+
+#include "qcom-ipq8064-v2.0.dtsi"
+
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	memory {
+		device_type = "memory";
+		linux,usable-memory = <0x41500000 0x1ea00000>;
+		reg = <0x40000000 0x20000000>;
+	};
+
+	cpus {
+		idle-states {
+			CPU_SPC: spc {
+				status = "disabled";
+			};
+		};
+	};
+
+	chosen {
+		bootargs-append = " console=ttyMSM0,115200n8 ubi.mtd=ubi ubi.mtd=art";
+	};
+};
+
+&qcom_pinmux {
+	mdio0_pins_active: mdio0_pins_active {
+		mux {
+			pins = "gpio0", "gpio1";
+			function = "gpio";
+			drive-strength = <2>;
+			bias-pull-down;
+			output-low;
+		};
+	};
+
+	phy_active: phy_active {
+		phy {
+			pins = "gpio6","gpio7";
+			function = "gpio";
+			drive-strength = <2>;
+			bias-pull-down;
+			output-high;
+		};
+	};
+
+	uart1_pins: uart1_pins {
+		mux {
+			pins = "gpio51", "gpio52", "gpio53", "gpio54";
```

Comment:
Are all four UART pins really used?

Reply:
There is a hint in the [u-boot source](https://github.com/clayface/U-boot-MR52-20200629/blob/8c6f4c2a84c050555e2dc7036b8287fd9381939d/U-boot.MR52/board/qcom/ipq806x_cdp/ipq806x_board_param.h#L357) that only `gpio51` & `gpio52` may be needed. I have tested and this seems to be all that's needed. Have adjusted the PR accordingly. Thanks!

Reply:
By default on this reference board series 4 pins are used as its supposed to be HSUART, however its possible that only 2 pins for regular UART are used